src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-card/info-card.component.scss

```diff
@@ -21,9 +21,13 @@ $card-font-max-size: 21px;
     padding-top: 40px !important;
 
     .card-title {
-      left: 0;
+      // title underline
+      border-bottom: 1px solid bd.$color-info-card-border;
```

Comment:
`$color-info-card-border` You may need to take care of this variable too.

Reply:
@nizamial09 Done, thanks!